pyproject.toml

```diff
@@ -48,6 +48,12 @@ platforms = ["linux-64"]
 [tool.pixi.pypi-dependencies]
 hepattn = { path = ".", editable = true }
 torch = { version = "==2.7.0", index = "https://download.pytorch.org/whl/test/cu126"}
+pybind11 = { version = "*" }
+energyflow = { version = "*" }
+pathos = { version = "*" }
+vector = { version = "*" }
+fastjet = { version = "*" }
+atlas-ftag-tools = { version = "*" }
```

Comment:
do we need all of these? I never needed `pybind11` so far, I guess some of the others are needed for the evaluation of the clic stuff but worth checking if we need them all. If we do we can move them further up with the other python dependencies (since we don't need a custom index for these)